blog/2025-01-28-ratify-1.4.0.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: ratify-v1.4.0-is-available
+title: Announcing Ratify v1.4.0 - Revocation Checking with CRL Support, Enhanced Helm Chart, and New Cloud Provider Support
+authors: [ratify]
+tags: [verification]
+---
+
+We are thrilled to announce the release of [Ratify v1.4.0](https://github.com/ratify-project/ratify/releases/tag/v1.4.0)! This milestone release introduces significant new features that enhance Ratify's capabilities as a trusted supply chain security tool. As always, we deeply appreciate the contributions from the community, which make these advancements possible.
+
+## New Features
+
+### Revocation Checking with Certificate Revocation List (CRL) Support
+   
+Security in supply chain verification is critical, and this release marks an important step forward with the addition of revocation checking. Ratify supports validating signing identity (certificate and certificate chain) revocation status using certificate revocation evaluation as per revocation setting in Notation trust policy. The CRL implementation uses Notation libraries and follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#crls). This ensures that revoked certificates are promptly identified and excluded from the trust chain, providing an extra layer of security for artifact verification workflows.
+See [Certificate Revocation Check (CRL) guidance](https://ratify.dev/docs/plugins/verifier/notation#certificate-revocation-check-crl) for details.
+
+### Enhanced Notation Trust Policy Attributes in Helm Chart
+
+To provide an out-of-the-box experience for users, this release adds more notation trust policy attributes to the `values.yaml` of the Ratify Helm Chart. This enhancement addresses three major scenarios outlined in [this issue](https://github.com/ratify-project/ratify/issues/1965), simplifying configuration and streamlining deployment processes for common use cases:
+
+- Organizational Signing Policies: Ratify now allows users to easily define and enforce organization-wide signing policies directly from the Helm Chart. This ensures that artifacts meet predefined security requirements before being trusted.
+
+- Customizable Configurations: Different environments, such as development, staging, and production, often require unique trust policies. With the updated Helm Chart, users can seamlessly customize configurations for each environment without manual adjustments.

```

Comment:
The current improvement is on helm chart, so it is mainly on deployment/production phase